message_definitions/v1.0/ras_a.xml

```diff
@@ -16,6 +16,67 @@
         <param index="2" label="X Position in the camera frame" minValue="0" increment="1">Position given by the number of pixels in X direction of the camera frame. Origin on the bottom left corner of the camera frame.</param>
         <param index="3" label="Y Position in the camera frame" minValue="0" increment="1">Position given by the number of pixels in Y direction of the camera frame. Origin on the bottom left corner of the camera frame.</param>
       </entry>
+      <entry value="36" name="MAV_CMD_DO_FOLLOW_SYS_ID" hasLocation="false" isDestination="false">
+        <description>Begin following a target</description>
+        <param index="1" label="System ID" minValue="1" maxValue="255" increment="1">System ID (of the FOLLOW_TARGET beacon).</param>
+        <param index="2">Reserved</param>
+        <param index="3">Reserved</param>
+        <param index="4" label="Altitude Mode" minValue="0" maxValue="2" increment="1">Altitude mode: 0: Keep current altitude, 1: keep altitude difference to target, 2: go to a fixed altitude above home.</param>
+        <param index="5" label="Altitude" units="m">Altitude above home. (used if mode=2)</param>
+        <param index="6">Reserved</param>
+        <param index="7" label="Time to Land" units="s" minValue="0">Time to land in which the MAV should go to the default position hold mode after a message RX timeout.</param>
+      </entry>
+      <entry value="37" name="MAV_CMD_DO_FOLLOW_POI" hasLocation="false" isDestination="false">
+        <description>Begin following a target specified by POI ID.  A POI Unique ID (UID) is a 64-bit field, and so to fully represent that UID, the big endian 64-bit POI UID is split into 2 words; param1: upper 32-bit word, param2: lower 32-bit word.</description>
+        <param index="1" label="POI UID Upper 32 Bits">The upper 32 bits of the UID from a POI_REPORT.</param>
+        <param index="2" label="POI UID Lower 32 Bits">The lower 32 bits of the UID from a POI_REPORT</param>
+        <param index="3" label="Mode Flags" enum="FOLLOW_POI_MODE_FLAGS">Bitmask of option flags.</param>
+        <param index="4" label="Altitude" units="mm or degE7">Altitude field, behavior to be interpreted according to Mode Flags.</param>
+        <param index="5" label="Azimuth" units="degE7">Azimuth field, behavior to be interpreted according to Mode Flags.</param>
+        <param index="6" label="Range" units="mm">Range field, behavior to be interpreted according to Mode Flags.</param>
+        <param index="7" label="Time to Return" units="s" minValue="0">Time to return in which the Autopilot should enter return to comms behavior. If 0, vehicle will immediaately enter return to comms behavior after losing comms with GCS</param>
+      </entry>
+      <entry value="38" name="MAV_CMD_DO_FOLLOW_NONE" hasLocation="false" isDestination="false">
```

Comment:
We don't need this. Follow commands should be followed by a mode change to "FOLLOW" mode. So this means that you can stop following a target by just using a [`MAV_CMD_DO_SET_MODE`](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_MODE) to the previous mode the vehicle was before. This also should be added to the description.

Reply:
Ok I think that workflow makes sense.